### DISCUSSION TITLE
The new version...hmm.

### DISCUSSION THREAD

**Discussion by BigHonkinDaddy** [powertab/powertabeditor#493] [General]
I hate to say it, but I'm seeing several issues with the new version: v2.0.20 64bit for Windows11 Home 23H2. The final straw was when I tried to use midi player and the app crashes everytime I press play on the midi player.

Other issues I noticed before the crash
- when windows desktop is in a dark mode, the toolbox is barely readable. My only recourse was to change my windows display to light mode and I don't like that.
- its impossible for me to delete a measure line.  

 I reinstalled 2.0 Beta  (v2.0.0-alpha19-0-g48acb0b)

**Comment by sbonaime** [comment]
I had to switch to last alpha version to remove bars !

**Comment by cameronwhite** [comment]
Could you please file bug reports for these issues? Including any crash logs you got from the midi issue

**Comment by cameronwhite** [comment]
The bar deletion issue should be fixed with 33935d81cb4120617a6f63c291e839fc8b929b4c - please give that build a test

**Comment by BigHonkinDaddy** [comment]
Yes, that's the fix I've been looking forward to, but alas, it wasn't working for me.

This weekend I'll reinstall and report defects on Git.


-------- Original message --------
From: Cameron White ***@***.***>
Date: 11/13/24 10:47 PM (GMT-05:00)
To: powertab/powertabeditor ***@***.***>
Cc: BigHonkinDaddy ***@***.***>, Author ***@***.***>
Subject: Re: [powertab/powertabeditor] The new version...hmm. (Discussion #493)


The bar deletion issue should be fixed with 33935d8<https://github.com/powertab/powertabeditor/commit/33935d81cb4120617a6f63c291e839fc8b929b4c> - please give that build a test
